README.md

```diff
@@ -10,34 +10,71 @@ Creating what we call [starter issues](http://hood.ie/blog/starter-issues.html)
 
 ### 💡💥❓ How things work
 
-Say I’m a Hoodie contributor and find a typo somewhere. Instead of fixing the issue directly in the master branch or creating a pull request, I create a new branch that is called something like first-timers-only-typo-in-title. GitHub will notify the "first-timers-only issue bot" about the new branch using Webhooks. Currently, the bot will see that a new branch was created starting with "first-timers-only" and it will create a new issue at https://github.com/hoodiehq/camp/issues/new based on a template and assign the first-timers-only and the up-for-grabs label. If the commit body can be used to add some context information. If left empty, the 🤔 What you will need to know section of the issue will simply say "Nothing :)". Otherwise the commit body will be used there.
+Say I’m a Hoodie contributor and find a typo somewhere. Instead of fixing the issue directly in the master branch or creating a pull request which is time-consuming, I can simply create a new branch that is called something like **first-timers-only-typo-in-title.** GitHub will then notify the "first-timers-only issue bot" about the new branch using Webhooks. The bot is listening to any new branch was created starting with **first-timers-only** and it will create a new issue at _https://github.com/hoodiehq/camp/issues/new_ with a template and assign the first-timers-only and the up-for-grabs label. The commit body can be used to add some context information and if left empty, the 🤔 **What you will need to know** section of the issue will simply say "Nothing :)".
```

Comment:
"The bot is listening to any new branch was created starting" - typo here, don't need was.

The issue will be created on the user's repo of choice. Hoodie is choosing the Camp repo, but that needs to be change by the user.